chef_master/source/windows.rst

```diff
@@ -251,8 +251,6 @@ For example:
 
    $ SCHTASKS.EXE /CREATE /TN ChefClientSchTask /SC MINUTE /MO 30 /F /RU "System" /RP /RL HIGHEST /TR "cmd /c \"C:\opscode\chef\embedded\bin\ruby.exe C:\opscode\chef\bin\chef-client -L C:\chef\chef-client.log -c C:\chef\client.rb\""
 
-Refer `Schedule a Task <https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc748993(v=ws.11)>`_ for more details.
```

Comment:
We probably need an updated version of this doc.

Reply:
I replaced it with a link to https://docs.microsoft.com/en-us/windows/win32/taskschd/schtasks